arrows/pdal/tests/test_pointcloud_io.cxx

```diff
@@ -50,6 +50,9 @@ TEST_F(pointcloud_io, save_geo_origin) {
     kv::landmark_map_sptr landmark_map =
         kv::read_ply_file(landmarks_path);
 
+    auto lgcs = kwiver::vital::local_geo_cs();
+    read_local_geo_cs_from_file(lgcs, geo_origin_path);
+
     auto io = new kwiver::arrows::pdal::pointcloud_io();
```

Comment:
This is a memory leak if there is no corresponding `delete` call.  However, you don't need this line at all if you follow the suggestion above to just use stand alone functions.